src/app/templates/activity/Activity.tsx

```diff
@@ -152,3 +135,53 @@ function mergeOperations(base?: IOperation[], toAppend: IOperation[] = []) {
   }
   return uniques;
 }
+
+interface IHandlerLoadMore {
```

Comment:
as I remember we decided to call `interfaces` as `ExampleInterface` instead of `IExample` (as it is on Temple Mobile)